extensions/amp-carousel/0.1/build-dom.js

```diff
@@ -287,9 +288,9 @@ export function buildDom(element) {
 /**
  * @param {Element} element
  * @return {string} The default title to use for the next button.
- * @param {{index?: string, total?: string}} options - The default title to use for the previous button.
+ * @param {{index: string, total: string}} options - The default title to use for the previous button.
  */
-export function getNextButtonTitle(element, options = {}) {
+export function getNextButtonTitle(element, options) {
```

Comment:
nit: If you end up with any other changes, may as well destructure the args here and `getPrevButtonTitle` like you do in `getButtonTitle`. If not, w/e
```suggestion
export function getNextButtonTitle(element, {index, total}) {
```